tasks/main.yml

```diff
@@ -21,11 +21,12 @@
       paths:
         - "vars"
 
-- name: Install Xfce desktop environment and TigerVNC server
+- name: Install TigerVNC server
   package:
     name:
       "{{ package_names }}"
 
+# Does this go here?  Can this be generalized?
```

Comment:
It would definitely be nice if we could find a more generalized solution here.  This was originally done because when the screen locked in a VNC session (initiated by a Guacamole user), that user does not know the VNC password to unlock it.

Info on light-locker is here: https://github.com/the-cavalry/light-locker

Reply:
My concern is that other distros may install a different screenlock utility.